src/blockdata.rs

```diff
@@ -110,10 +112,10 @@ impl Block {
 
     /// Returns block hash
     pub fn hash(&self) -> hash::SHA256Hash {
-        let header = if self.0[Self::PROOF_POSITION] == 0 {
-            &self.0[..(Self::PROOF_POSITION + 1)] // length byte
+        let header = if self.0[Self::AGG_PUBKEY_POSITION] == 0 {
```

Comment:
This implementation looks something strange.
It must check whether the block includes proof and aggregated public key.

Reply:
In this function, only the name of the constants is changed, because it is not used to generate a block signature.

Reply:
But the return value is wrong as block hash.

Reply:
Now, function `hash()` is used only in logging.
Fixing `hash()` is not out of the scope of this PR.

I think it is better to create another issue and fix it.
Using rust-tapyrus to calculate block hash may be a good way.

Reply:
That make sense. fixing `hash()` is out of scope.

I confused because the constant name was changed. 
We don't need to change the name. 105 is the position of proof length field.
but is is a little issue we can ignore it.